bpf/lib/common.h

```diff
@@ -422,7 +422,6 @@ struct ct_state {
 	__u16 loopback:1,
 	      reserved:15;
 	__be16 orig_dport;
-	__be16 proxy_port;
```

Comment:
Should this be replaced with padding?

Reply:
It's not stored in any maps so I did not add padding. Any particular reason for the padding?